lib/base-compiler.js

```diff
@@ -355,6 +369,14 @@ Compile.prototype.couldSupportASTDump = function (options, version) {
     return false;
 };
 
+Compile.prototype.supportCFGGeneration = function(compilerVersion){
+    return (compilerVersion.indexOf("g++") === 0);
+};
+
+Compile.prototype.filtersAreSet = function(filters){
```

Comment:
This function appears to be unused. If that's the case, please remove, else add some comments explaining what it's for please.

Reply:
done